server/game/cards/02.4-NMG/RedCloaks.js

```diff
@@ -5,11 +5,9 @@ class RedCloaks extends DrawCard {
         this.action({
             title: 'Move 1 gold from your gold pool to this card',
             limit: ability.limit.perPhase(1),
-            condition: () => this.controller.gold > 0,
+            condition: () => this.controller.hasEnoughGold(1),
```

Comment:
Do we have making actions support costs on the todo list anywhere?

Reply:
They do, but Red Cloaks has no cost associated with its ability. Gold needs to be checked to make sure triggering the ability has the potential to change the game state.